packages/docusaurus-theme-classic/src/theme/CodeBlock/styles.module.css

```diff
@@ -9,7 +9,6 @@
   margin-bottom: var(--ifm-leading);
   border-radius: var(--ifm-global-radius);
   box-shadow: var(--ifm-global-shadow-lw);
-  overflow: hidden;
```

Comment:
apparently I added this in https://github.com/facebook/docusaurus/pull/5215, don't remember why 😅 hope it's safe to remove

Reply:
I realized what `overflow: hidden` does... Because if we set border radius on the parent container but child is still a regular rectangle, the child overflows and makes the code block still appear as a sharp-edged rectangle

Reply:
Ah yes, I remember that was related to border-radius  now 😅 it was simpler to add radiuses on the container than on the individual items, particularly when the live codeblock playground/result positions are configurable